modules/repository-azure/build.gradle

```diff
@@ -72,11 +73,11 @@ dependencies {
   api "org.reactivestreams:reactive-streams:${versions.reactiveStreams}"
 
   implementation project(":modules:transport-netty4")
+  implementation("org.slf4j:slf4j-api:${versions.slf4j}")
```

Comment:
The `com.azure.core` module requires *transitive* org.slf4j, so it must be present at both compile time and run time.